apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.resolver.ts

```diff
@@ -1,17 +1,18 @@
-import { Inject } from '@nestjs/common'
+import { Inject, NotFoundException, forwardRef } from '@nestjs/common'
 import { Query, Resolver, Args, Mutation } from '@nestjs/graphql'
 
 import type { Logger } from '@island.is/logging'
 import { LOGGER_PROVIDER } from '@island.is/logging'
 
-import { Authorize, Role } from '../auth'
+import { Authorize, Role, CurrentUser, User } from '../auth'
 import { VehicleModel } from '../vehicle'
 import {
   RecyclingRequestModel,
   RecyclingRequestTypes,
   RecyclingRequestResponse,
 } from './recyclingRequest.model'
 import { RecyclingRequestService } from './recyclingRequest.service'
+import { SamgongustofaService } from '../samgongustofa/samgongustofa.service'
```

Comment:
Don't dig into the modules for imports ... index.ts is the `api` of the module and things should only be imported from there.
Keep the imports clean and move it upwards below `../vehicle`

Reply:
`*.module` are the only exceptions.